src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Commands/Analyze/ConsoleAnalyze.cs

```diff
@@ -18,25 +19,29 @@ public class ConsoleAnalyze : IAppCommand
         private readonly IEnumerable<IAnalyzeResultProvider> _providers;
         private readonly IAnalyzeResultWriter _writer;
         private readonly IExtensionProvider _extensionProvider;
+        private readonly IOptions<AnalysisOptions> _options;
 
         public ConsoleAnalyze(
             IEnumerable<IAnalyzeResultProvider> analysisProviders,
             IUpgradeContextFactory contextFactory,
             IUpgradeStateManager stateManager,
             IAnalyzeResultWriter writer,
-            IExtensionProvider extensionProvider)
+            IExtensionProvider extensionProvider, IOptions<AnalysisOptions> options)
         {
             _providers = analysisProviders ?? throw new ArgumentNullException(nameof(analysisProviders));
             _writer = writer ?? throw new ArgumentNullException(nameof(writer));
             _extensionProvider = extensionProvider ?? throw new ArgumentNullException(nameof(extensionProvider));
+            _options = options ?? throw new ArgumentNullException(nameof(options));
             _contextFactory = contextFactory ?? throw new ArgumentNullException(nameof(contextFactory));
             _stateManager = stateManager ?? throw new ArgumentNullException(nameof(stateManager));
+
+           // _contextFactory.CreateContext
         }
 
         public async Task RunAsync(CancellationToken token)
         {
             using var context = await _contextFactory.CreateContext(token);
-
+            
```

Comment:
nit: this will cause a warning in VS due to extra whitespace. We fail the CI build on any warnings so make sure to pay attention to that window while working (we intentionally don't raise errors while developing so it doesn't block you, but do at CI to ensure we have consistent style)

Reply:
Resolved